source/AI.cpp

```diff
@@ -1765,20 +1765,17 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition, const
 	
 	bool shouldReverse = false;
 	dp = targetPosition - StoppingPoint(ship, targetVelocity, shouldReverse);
-	if(shouldReverse && dp.Length() < radius)
-	{
-		// We can directly use the reverse thrusters to stop at the target.
-		command |= Command::BACK;
-		return false;
-	}
 
-	bool isFacing = (dp.Unit().Dot(angle.Unit()) > .8);
+	bool isFacing = (dp.Unit().Dot(angle.Unit()) > .95);
 	if(!isClose || !isFacing)
```

Comment:
When the `if(should reverse...)` bit was added above, this line was changed from `if(!isClose || (!isFacing && !shouldReverse))` to what it is now. Should this line be changed back to what it was?

Reply:
Yeah, I think this needs readded, as otherwise you could do a command.SetTurn here and down on line 1776.